core/store/migrate/migrations/0168_drop_persitent_config_storage.sql

```diff
@@ -0,0 +1,92 @@
+-- +goose Up
+
+-- Drop nodes tables
+DROP TABLE IF EXISTS evm_nodes;
+DROP TABLE  IF EXISTS solana_nodes;
+DROP TABLE  IF EXISTS starknet_nodes;
+
+-- Drop cfg column from chains
+ALTER TABLE  IF EXISTS evm_chains DROP COLUMN cfg;
+ALTER TABLE  IF EXISTS solana_chains DROP COLUMN cfg;
+ALTER TABLE  IF EXISTS starknet_chains DROP COLUMN cfg;
```

Comment:
Do these serve any purpose besides satisfying foreign key constraints? I thought we would be able to drop them.

Reply:
oh, i'm not sure, i read the tix as only dropping a field from them. let me double check

Reply:
Oh yeah I wrote it subtly as 'at least remove...' :facepalm:

Reply:
updated to drop the chains tables and all the FKs

Reply:
reduced scope to only deal with node tables as discussed